index.html

```diff
@@ -1885,6 +1887,40 @@ <h2>
           shipping option. <a>User agents</a> MAY indicate pending fields in
           the user interface for the payment request.
         </dd>
+        <dt>
+          <dfn>type</dfn> member
+        </dt>
+        <dd data-tests="PaymentItem/type_member.https.html">
+          A <a>PaymentItemType</a> enum value or null. A user agent MAY use the
+          value of <a>type</a> to assist in the presentation of
+          <a>PaymentItem</a> by, for example, visually grouping types together
+          or other otherwise distinguishing them from other types (or from
+          items that have no associated type).
+        </dd>
+      </dl>
+    </section>
+    <section data-dfn-for="PaymentItemType">
+      <h2>
+        <dfn>PaymentItemType</dfn> enum
+      </h2>
+      <pre class="idl">
+        enum PaymentItemType {
+          "tax"
+        };
+      </pre>
+      <p>
+        The <a>PaymentItemType</a> serves to categorize a <a>PaymentItem</a>
+        into particular types.
+      </p>
+      <dl>
+        <dt>
+          "<dfn>tax</dfn>"
+        </dt>
+        <dd>
+          Indicates that the corresponding <a>PaymentItem</a> represents a form
+          of tax (e.g., sales tax, goods and services tax, value added tax,
+          etc.).
```

Comment:
Would it be useful to say something like "(The developer can use the `label` member to be more specific.)"? I guess in that case I'd move the examples outside of the parentheses so there aren't too many parentheses.

Reply:
Yeah, but I think the developer bit would probably be better in `type` member above, so will add it there. I'll remove the parens.